### DISCUSSION TITLE
make launch failed

### DISCUSSION THREAD

**Discussion by wayneyo12** [carla-simulator/carla#7883] [Q&A]
CARLA version: 0.9.15
Platform/OS: Ubuntu 20.0.4,
![image](https://github.com/carla-simulator/carla/assets/73013282/d3c506ef-6cec-4b5a-8851-fc140291c9cf)

Problem you have experienced:  
I downloaded this branch from https://github.com/EpicGames-Mirror-A/UnrealEngine/tree/4.26, and then followed each step according to the official Carla instructions. However, I encountered the following error when running make launch:
![makelaunch_error](https://github.com/carla-simulator/carla/assets/73013282/d2100411-7ca5-430c-8bad-57b62ed1328e)

I want to know where I might have gone wrong. Is it because the official Epic repository doesn't have a special branch for Carla, leading to missing files, or did I make some mistakes during the download process?

**Comment by Daraan** [comment]
~~I do not know if anything changes but the official repository has a "carla" branch that you have to use.~~

~~`git clone --depth 1 -b carla https://github.com/CarlaUnreal/UnrealEngine.git ~/UnrealEngine_4.26`~~

Never mind, sorry there seams to be ongoing issue, check the issues and other discussions about the changes on Epic

**Reply by wayneyo12** [reply]
Thank you sir, so is this an official issue with Epic Games?
